controllers/provisioners/ekscloudformation/strategy.go

```diff
@@ -68,8 +126,14 @@ func (ctx *EksCfInstanceGroupContext) processCRDStrategy() error {
 		return err
 	}
 
+	addAnnotation(customResource, OwnershipAnnotationKey, "true")
```

Comment:
If this annotation is meant to convey "ownership" by another resource or your controller might be better to use a Kubernetes ownerreference instead. https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents

If this is meant to convey some management association you could use a best-practice label here e.g., https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels
`app.kubernetes.io/managed-by: instancemgr.orkaproj.io`

Reply:
You might want both annotation and label here depending on your use-case.

Reply:
since CRD strategy is generic, I don't know what CR the user will submit as part of upgrade, hence I cannot foresee if `ownerReference` will be supported by the controller.
I will look into changing the annotation key/value to best practice